### PR TITLE
tests/main: add test case to verify core snap conf option service.ssh.disabled

### PR DESCRIPTION
**Pull request by morphis**
This requires https://code.launchpad.net/~morphis/core-snap/add-configure-hook-sshd-service/+merge/315203 to be merged for the core snap.

### CODE REVIEW

File: tests/main/core-configure-sshd/task.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+restore: |
+    umount /lib/systemd/system/ssh.service || true
+
+execute: |
+    # Default value should be always 'false' on a fresh system with
+    # no extra configuration coming from the gadget snap
+    test `snap get core service.sshd.disabled` = false
+    test ! -e /etc/ssh/sshd_not_to_be_run
+
+    cat << EOF > /tmp/fake-ssh.service
```
Comment (mvo5) [inline tests/main/core-configure-sshd/task.yaml]: Nice trick!

File: tests/main/core-configure-sshd/task.yaml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+    systemctl daemon-reload
+
+    snap set core service.sshd.disabled=true
+    test `snap get core service.sshd.disabled` = true
+    test -e /etc/ssh/sshd_not_to_be_run
+    test -e /tmp/sshd-stopped
+    rm /tmp/sshd-stopped
+
+    snap set core service.sshd.disabled=false
```
Comment (mvo5) [inline tests/main/core-configure-sshd/task.yaml]: (nitpcik) you could test for invalid values for service.sshd.disabled here too

Comment (morphis) [inline tests/main/core-configure-sshd/task.yaml]: Let me do that.

**Comment by morphis** [comment]
Closing for now as it doesn't work as I thought.

**Pull request closed without merging**